articles/load-balancer/quickstart-create-basic-load-balancer-powershell.md

```diff
@@ -21,8 +21,6 @@ ms:custom: seodec18
 
 This quickstart shows you how to create Basic Load Balancer using Azure PowerShell. To test the load balancer, you deploy two virtual machines (VMs) running Windows server and load balance a web app between the VMs.
 
-[!INCLUDE [updated-for-az](../../includes/updated-for-az.md)]
```

Comment:
This line includes a piece of information that the article was updated to CLI, and it's not true, the article is entirely written in ARM